apps/testing/lib/temp_env.ex

```diff
@@ -27,6 +27,11 @@ defmodule Temp.Env do
         end)
 
         on_exit(fn ->
+          unquote(entries)
+          |> Enum.each(fn %{app: app, key: key} ->
+            Application.delete_env(app, key)
+          end)
```

Comment:
This is to prevent adding multiple entries being added b/c of keyword list behavior?